docs/client/Account.md

```diff
@@ -3,10 +3,10 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Description** | **string** |  | [optional] 
 **Href** | **string** |  | [optional] 
 **Id** | **string** |  | [optional] 
 **Name** | **string** |  | 
+**Description** | **string** |  | [optional] 
 **Parent** | [***Link**](Link.md) |  | [optional] 
 **SupportedConnectionGroups** | [**[]Link**](Link.md) |  | [optional] 
```

Comment:
Is there any way to get these properties to be output in the same order each run or are we causing this by moving things around?

Reply:
It looks like it was alphabetically sorted and now is not.

Reply:
Yeah, I think I had previously run `python -m json.tool` which sorts the schema alphabetically.
Let me generate the source. There were a could of bug fixes needed anyway.

Reply:
Should be updated now.